src/components/Hero/Hero.module.css

```diff
@@ -1,5 +1,6 @@
 .hero {
-	--hero-heading: var(--medium-primary-color);
+	--hero-heading: var(--dark-red);
```

Comment:
is there a reason why this CSS variable was renamed? We acn just use the same variables, but change the value in the global.css file